docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,34 @@
+version: "3"
+
+volumes:
+  db-data:
+    external: false
+
+services:
+  proxy:
+    extends:
+      file: docker-compose.common.yml
+      service: proxy
+
+
+  database:
+    extends:
+      file: docker-compose.common.yml
+      service: proxy
+    volumes:
+      - db-data:/var/lib/postgresql/db-data
```

Comment:
@nirgn975 In relation to [comment above](https://github.com/hasadna/open_pension/pull/351/files#r134669874) i'm trying to mount the `db-data volume` to the place where the postgres containers database is. so it will persist in the developers host machine.